ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -813,13 +813,9 @@ def arrays_to_numpy(x):
     if frontend_config.isscalar(frontend_ret):
         frontend_ret_np_flat = [frontend_config.to_numpy(frontend_ret)]
     else:
-        # tuplify the frontend return
-        if not isinstance(frontend_ret, tuple):
-            frontend_ret = (frontend_ret,)
-        frontend_ret_idxs = ivy.nested_argwhere(
-            frontend_ret, frontend_config.is_native_array
+        frontend_ret_flat = flatten_frontend(
+            ret=ret, backend=backend_to_test, frontend_array_fn=create_frontend_array
```

Comment:
I think we shouldn't rely on using `create_frontend_array` as not all frontends have it, also we should modify flatten_frontend to use the frontend config in that case. similarly for `test_frontend_method`.